Odata-docs/webapiauth/how-permissions-are-applied.md

```diff
@@ -0,0 +1,119 @@
+---
+title : "How Permissions are Applied"
+
+author: habbes
+ms.author: habbes
+ms.date: 8/23/2020
+---
+# How permissions are applied
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)]
+
```

Comment:
Just confirming that this only applies to v7

Reply:
Yeah, just 7.